agate/agate/tests/test_api_endpoints.py

```diff
@@ -0,0 +1,82 @@
+from rest_framework import status
+from rest_framework.test import APITestCase
+from agate.models import IngestionAttempt
+from agate.caching import TokenCache
+from django.urls import reverse
+import hashlib
+
+auth = "my_token"
+
+
+class IngestionAttemptAPITests(APITestCase):
+
+    def setUp(self):
+        # Setup: Create some IngestionAttempt instances for testing
+        IngestionAttempt.objects.create(uuid="user1",
+                                        is_published=True,
+                                        project="project",
+                                        site="here",
+                                        is_test_attempt=False)
+        IngestionAttempt.objects.create(uuid="user2",
+                                        is_published=True,
+                                        project="project",
+                                        site="here",
+                                        is_test_attempt=False)
+
+        TokenCache.objects.create(
+            projects_output='{"data": [{"project":"project"}]}',
+            site_output="here",
+            token_hash=hashlib.sha256(auth.encode("utf-8")).hexdigest()
+        )
```

Comment:
I'm not too precious about these details but note this mixes two indentation styles,
```py
function(a,
         b)
```
and
```py
function(
    a,
    b,
)
```
(That last trailing comma is optional.)